int32 MotorSpeed
int32 Mode
int32 Direction
int32 Supersonic

int32 ax
int32 ay
int32 az

int32 alphax
int32 alphay
int32 alphaz

int32 Bx
int32 By
int32 Bz

int32 thetax
int32 thetay
int32 thetaz

int32 BatteryVoltage
int32 MotorTemperature
